gammapy/datasets/map.py

```diff
@@ -2061,10 +2061,10 @@ def alpha(self):
             Alpha map
         """
         with np.errstate(invalid="ignore", divide="ignore"):
-            alpha = self.acceptance / self.acceptance_off
+            data = self.acceptance.quantity / self.acceptance_off.quantity
+        data = np.nan_to_num(data)
```

Comment:
Is it safe to replace the NaN by 0? Indeed, with a zero mathematical operations like interpolations are permitted, whereas a Nan mentions explicitly that the NaN bins should not be used!

Reply:
It is necessary here otherwise, when summing over `alpha * counts_off` you will get a `nan` result.

No interpolations are made with `alpha`.